test/testutil.h

```diff
@@ -649,4 +649,10 @@ X509 *load_cert_der(const unsigned char *bytes, int len);
 STACK_OF(X509) *load_certs_pem(const char *file);
 X509_REQ *load_csr_der(const char *file, OSSL_LIB_CTX *libctx);
 
+/*
+  moved from ./include/crypto/asn1.h  
+  time_t ossl_asn1_string_to_time_t(const char *asn1_string);
+*/
```

Comment:
I don't think this comment is really necessary.

Reply:
Yeah, it is also wrongly formatted. Please just drop it.

Reply:
If this block (which has trailing whitespace on line 653) is deleted, I can approve this PR